.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
Ok. I run some tests and, in fact, if `main` equals `./lib/index.js`, then the module can be correctly imported / required, otherwise, it fails. I had my doubts about that.

About the `files` properties, I wasn't aware of what it is meant to do.

So, I should change that part of package.json to something like this...

```json
    "main":        "./lib/index.js",
    "files":       ["./lib"],
```

... and remove .npmignore?